### PR TITLE
room module

### PR DESCRIPTION
**Pull request by shivangi-gupta123**
Room module of admin Dashboard - get all room api , table component, loader implement, up down image, eye image for display the last survival, highest kill, second Win , Third Win, deleted  functionality 

https://app.clickup.com/t/85ztkt5dj - click up link

### CODE REVIEW

File: src/app/adminDashboard/room/page.tsx [hunk 1]
```diff
@@ -1,80 +1,133 @@
 'use client';
-import React, {useState, useEffect} from 'react';
+import React, { useState, useEffect } from 'react';
 import styles from '../../../styles/Dashboard.module.scss';
-import {Navbar} from '../../../Components/Navbar/Navbar';
-import TableData, {StudentProfile} from '../../../Components/Table/Table';
+import { Navbar } from '../../../Components/Navbar/Navbar';
+
```
Comment (rajeshdeveloper99) [inline src/app/adminDashboard/room/page.tsx]: remove white space @shivangi-gupta123

File: src/app/adminDashboard/room/page.tsx [hunk 2]
```diff
@@ -1,80 +1,133 @@
...
+  const getAllTournaments = async () => {
+    const tokens = localStorage.getItem('jwtToken');
+    const tournamentResponse = await sendRequest('/room/rooms', {
+      method: 'GET',
+      headers: { 'Authorization': `Bearer ${tokens}` }
+    });
+
+    setRoomData(tournamentResponse.data);
+
```
Comment (rajeshdeveloper99) [inline src/app/adminDashboard/room/page.tsx]: remove white space @shivangi-gupta123

File: src/app/adminDashboard/room/page.tsx [hunk 3]
```diff
@@ -1,80 +1,133 @@
...
+    console.log("id__________", _id)
+    setIsLoading(true);
+    try {
+      const tokens = localStorage.getItem('jwtToken');
+      console.log(tokens)
+      const response = await sendRequest(`/room/rooms/${_id}`, {
+        method: 'delete',
+        headers: { 'Authorization': `Bearer ${tokens}` }
+      });
+      // const updatedData = roomData.filter(data => data.roomId !== _id);
```
Comment (rajeshdeveloper99) [inline src/app/adminDashboard/room/page.tsx]: remove unused code @shivangi-gupta123

File: src/app/adminDashboard/room/page.tsx [hunk 4]
```diff
@@ -1,80 +1,133 @@
...
+      // setRoomData(updatedData);
+      getAllTournaments();
+      if (response) {
+
+        const success = response.data.message;
+        toast.success(success);
+
```
Comment (rajeshdeveloper99) [inline src/app/adminDashboard/room/page.tsx]: remove white space @shivangi-gupta123

File: src/app/adminDashboard/room/page.tsx [hunk 5]
```diff
@@ -1,80 +1,133 @@
...
+      }
+
+    } catch (error) {
+      console.error('Error deleting room:', error);
```
Comment (rajeshdeveloper99) [inline src/app/adminDashboard/room/page.tsx]: remove console code @shivangi-gupta123

File: src/app/adminDashboard/room/page.tsx [hunk 6]
```diff
@@ -1,80 +1,133 @@
...
+
+    } finally {
+      setIsLoading(false);
+    }
   };
 
+
```
Comment (rajeshdeveloper99) [inline src/app/adminDashboard/room/page.tsx:68]: remove whiet space @shivangi-gupta123

File: src/app/adminDashboard/room/page.tsx [hunk 7]
```diff
@@ -1,80 +1,133 @@
...
+    'Room Id',
+    'Password',
+    'Game Name',
+    'Game Type',
+    'Map Type',
+    // 'MapImg',
+    'Version',
+    'Time',
+    'Date',
+    // 'Last Servival',
```
Comment (rajeshdeveloper99) [inline src/app/adminDashboard/room/page.tsx]: reomve commented code @shivangi-gupta123

File: src/app/adminDashboard/room/page.tsx [hunk 8]
```diff
@@ -1,80 +1,133 @@
...
-              />
-              {/* <Pagination
-                currentPage={currentPage}
-                totalPages={Math.ceil(
-                  transformedStudentData.length / rowPerPage,
-                )}
-                onPageChange={onPageChange}
-              /> */}
+              <h1 className={styles.heading}>Welcome to Admin Dashboard</h1>
+              {/* <BtnDashboard /> */}
```
Comment (rajeshdeveloper99) [inline src/app/adminDashboard/room/page.tsx]: fix name converstation @shivangi-gupta123

File: src/app/auth/login/page.tsx [hunk 9]
```diff
@@ -121,7 +121,7 @@ function Login(): React.JSX.Element {
       const decodedToken: any = decodeJWt(token);
       console.log('tokennnn', decodedToken.role.role);
```
Comment (rajeshdeveloper99) [inline src/app/auth/login/page.tsx:122]: again remove console @shivangi-gupta123

File: src/app/auth/login/page.tsx [hunk 10]
```diff
@@ -121,7 +121,7 @@ function Login(): React.JSX.Element {
...
       if (decodedToken.role.role === 'admin') {
-        router.push('/adminDashboard');
+        router.push('/adminDashboard/room');
       } else if (decodedToken.role.role === 'user') {
         router.push('/userDashboard');
         // router.push(configData.web.cominSoonUrl)
```
Comment (rajeshdeveloper99) [inline src/app/auth/login/page.tsx:127]: remove unused code @shivangi-gupta123

File: src/styles/Dashboard.module.scss [hunk 11]
```diff
@@ -25,6 +28,11 @@
     margin-left: 8px !important;
```
Comment (rajeshdeveloper99) [inline src/styles/Dashboard.module.scss:27]: why use important  @shivangi-gupta123

File: src/styles/Dashboard.module.scss [hunk 12]
```diff
@@ -831,3 +839,131 @@ img.slides_image {
   background: #ff7a00;
```
Comment (rajeshdeveloper99) [inline src/styles/Dashboard.module.scss]: use variables code in background-color @shivangi-gupta123

File: src/styles/Dashboard.module.scss [hunk 13]
```diff
@@ -831,3 +839,131 @@ img.slides_image {
...
+
+.overlay {
+  background: rgba(49, 49, 49, 0.8);
+}
+.modalcontent {
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  // line-height: 1.4;
```
Comment (rajeshdeveloper99) [inline src/styles/Dashboard.module.scss]: reomve unused code @shivangi-gupta123

File: src/styles/TableData.module.scss [hunk 14]
```diff
@@ -3,25 +3,44 @@
...
 
 .filter {
   display: flex;
-  align-items: flex-start;
-  width: auto;
+  align-items: center;
+  font-size: 12px;
+  font-weight: 700;
+  white-space: nowrap;
+
```
Comment (rajeshdeveloper99) [inline src/styles/TableData.module.scss]: remove unused code and commented code @shivangi-gupta123

File: src/styles/TableData.module.scss [hunk 15]
```diff
@@ -33,7 +52,8 @@
 }
 
 .table_rowdata {
-  background-color: #f5f5f5;
+  @include tdbody;
+  font-weight: 800;
 }
 
 .table_icon {
```
Comment (rajeshdeveloper99) [inline src/styles/TableData.module.scss:58]: fix this code @shivangi-gupta123

File: src/styles/TableData.module.scss [hunk 16]
```diff
@@ -73,3 +95,236 @@
 .table_Studenticon img {
```
Comment (rajeshdeveloper99) [inline src/styles/TableData.module.scss:93]: why use img tag for className  @shivangi-gupta123

File: src/styles/TableData.module.scss [hunk 17]
```diff
@@ -73,3 +95,236 @@
...
+  font-family: Century Gothic;
+  line-height: 36.78px;
+}
+.check_model {
+  background: white;
+  height: auto;
+  width: 30%;
+  border-radius: 15px;
+  @media (max-width: $breakpoint-xSmall) {
+    width: 90% !important;
```
Comment (rajeshdeveloper99) [inline src/styles/TableData.module.scss:115]: remove important @shivangi-gupta123

File: src/styles/TableData.module.scss [hunk 18]
```diff
@@ -73,3 +95,236 @@
...
+  }
+  @media (max-width: $breakpoint-xs) {
+    width: 90% !important;
```
Comment (rajeshdeveloper99) [inline src/styles/TableData.module.scss:118]: remove important @shivangi-gupta123

File: src/styles/TableData.module.scss [hunk 19]
```diff
@@ -73,3 +95,236 @@
...
+  background-color: rgba(0, 0, 0, 0.18);
+}
+
+.inner_specter_cls {
+  display: flex;
+  justify-content: space-between;
+  padding: 4px 26px;
+}
+.main_form_btn {
+  background: #ff7a00;
```
Comment (rajeshdeveloper99) [inline src/styles/TableData.module.scss]: remove  hardcode color code use varibles scss @shivangi-gupta123

File: src/styles/TableData.module.scss [hunk 20]
```diff
@@ -73,3 +95,236 @@
...
+    width: 50%;
+  }
+}
+.main_pop_cls,
+.main_del_sec {
+  @include spectator_model;
+}
+
+.main_del_sec {
+  background-color: rgba(0, 0, 0, 0.18);
```
Comment (rajeshdeveloper99) [inline src/styles/TableData.module.scss]: remove hardcode color code use varibles scss @shivangi-gupta123

File: src/styles/TableData.module.scss [hunk 21]
```diff
@@ -73,3 +95,236 @@
...
+  @media (max-width: $breakpoint-xs) {
+    margin: 8px 0px;
+  }
+}
+.room_id {
+  color: #003554 !important;
+  font-weight: 700;
+  font-size: 14px;
+  line-height: 17.17px;
+  font-family: Century Gothic;
```
Comment (rajeshdeveloper99) [inline src/styles/TableData.module.scss:188]: remove hardcode font-family  use @shivangi-gupta123

File: src/styles/TableData.module.scss [hunk 22]
```diff
@@ -73,3 +95,236 @@
...
+.form_spectator_cls {
+  padding: 28px 48px;
+  height: 500px;
+  overflow-y: scroll;
+}
+button#btn_cls_st {
+  background: #ff7a00;
+}
+.roombutton {
+  background: #ff7a00 !important;
```
Comment (rajeshdeveloper99) [inline src/styles/TableData.module.scss]: reomve important @shivangi-gupta123

**Review by rajeshdeveloper99 — COMMENTED** [review]
1. remove unused code 
2. remove important from scss file 
3. remove hardcoded color code 
4. reomve white spaces
5. remove console log

**Review by vijayg963TG — APPROVED** [review]

**Review by vijayg963TG — APPROVED** [review]

**Review by vijayg963TG — APPROVED** [review]

**Pull request merged**